Dockerfile.build

```diff
@@ -0,0 +1,27 @@
+## Builder part is quite heavy as it depends on the Go toolchain
+FROM golang:1.11-alpine AS builder
+LABEL maintainer="BOSCoin Developers <devteam@boscoin.io>"
+
+
+RUN apk add --no-cache git openssh gcc musl-dev linux-headers
+RUN go get github.com/ahmetb/govvv
+
+#golang:alpine set $GOPATH to `/go`
+COPY ./ /sebak-build
+WORKDIR /sebak-build
+
+## Note that we do not get the dependencies anew
+## We carry over whatever is in `vendor`, so the user MUST run `dep ensure` in their local copy
+## This make building the container orders of magnitude faster (`dep ensure` is extremely slow),
+## greatly reduce the container's size, and gives more control to the user as to what is tested
+## (one can replace a dependency, if needed).
+## Otherwise, install dep and dependencies.(e.g Docker Hub automated build)
+RUN if [ ! -d vendor ]; then go get github.com/golang/dep/cmd/dep; dep ensure -v; fi
```

Comment:
This is outdated

Reply:
I was worked on master.
Is it updated?

Reply:
Look at what's in master

Reply:
I will handle this Fail. https://circleci.com/gh/bosnet/sebak/8902?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

It looks like due to this.